Brocfile.js

```diff
@@ -0,0 +1,35 @@
+/* global require, module */
+
+var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
+var fs = require('fs');
+
+// TODO(azirbel): Use something more legit
+function htmlEntities(str) {
+  return String(str).replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;').replace(/"/g, '&quot;');
+}
+
+var app = new EmberAddon({
+  /*
+   * Replace patterns. We use this to replace strings such as:
+   * @@{controllers/file.js}
+   * With the content of those files.
+   */
+  replace: {
+    files: [
+      '**/*.js'
+    ],
+    patterns: [{
+      match: /@@{[^}]*}/g,
+      replacement: function(matchedText) {
+        filename = matchedText.slice(3, -1);
```

Comment:
I suppose this is fine, but can't you capture the inner text using a regex group and consume that? IMHO that would be cleaner/more readable code.